x-pack/plugins/stack_connectors/public/connector_types/servicenow_itsm/types.ts

```diff
@@ -7,6 +7,11 @@
 
 import type { ExecutorSubActionPushParamsITSM } from '../../../server/connector_types/lib/servicenow/types';
 
+export enum EventAction {
+  Trigger = 'trigger',
+  Resolve = 'resolve',
```

Comment:
Very trivial nit: looks like the convention is using all capital in enums.

Reply:
Updated in https://github.com/elastic/kibana/pull/177026/commits/c541c06fd197336bfc44ed0ee1d084b4cc39822f